cache-config/t3c-apply/t3c-apply.go

```diff
@@ -69,6 +69,9 @@ func main() {
 	} else if cfg == (config.Cfg{}) { // user used the --help option
 		os.Exit(Success)
 	}
+	if !lock.GetLock("/var/run/t3c.lock") {
```

Comment:
Does this need a `if !cfg.ReportOnly {` check like the old code had, to allow running in Report Mode while syncds runs?

Reply:
Not in my opinion.  The idea with the locking is limit t3c processes and logins to TO.  Why allow a t3c process running in ReportOnly, it still has to login to TO.